reference/python-sdk/v3.2.md

```diff
@@ -0,0 +1,1472 @@
+
+
+
+* **Returns**
+
+    Document response from REST API
+
+
+
+* **Return type**
+
+    dict
+
+
+
+* **Raises**
+
+    `InvalidCredentialsException`, `TooManyRequestsException`, `LimitExceededException`, `requests.exception.RequestException`
+
+
+
+#### create_prediction(document_id: str, model_id: str, \*, max_pages: Optional[int] = None, auto_rotate: Optional[bool] = None)
+Create a prediction on a document using specified model, calls the POST /predictions endpoint.
+
+```python
+>>> from las.client import Client
+>>> client = Client()
+>>> client.create_prediction(document_id='<document id>', model_id='<model id>')
+```
+
+
+* **Parameters**
+
+    
+    * **document_id** (*str*) – Id of the document to run inference and create a prediction on
+
+
+    * **model_id** (*str*) – Id of the model to use for inference
+
+
+    * **max_pages** (*int*) – Maximum number of pages to run predictions on
+
+
+    * **auto_rotate** (*bool*) – Whether or not to let the API try different rotations on the document when running predictions
+
+
+
+* **Returns**
+
+    Prediction response from REST API
+
+
+
+* **Return type**
+
+    dict
+
+
+
+* **Raises**
+
+    `InvalidCredentialsException`, `TooManyRequestsException`, `LimitExceededException`, `requests.exception.RequestException`
+
+
+
+#### create_secret(data: dict, \*\*optional_args)
+Creates an secret handle, calls the POST /secrets endpoint.
+
+```python
+>>> from las.client import Client
+>>> client = Client()
+>>> data = {'username': '<username>', 'password': '<password>'}
+>>> client.create_secret(data, description='<description>')
+```
+
+
+* **Parameters**
+
+    
+    * **data** (*str*) – Dict containing the data you want to keep secret
+
+
+    * **name** (*str*) – Name of the secret
+
+
+    * **description** (*str*) – Description of the secret
+
+
+
+* **Returns**
+
+    Secret response from REST API
+
+
+
+* **Return type**
+
+    dict
+
+
+
+* **Raises**
+
+    `InvalidCredentialsException`, `TooManyRequestsException`, `LimitExceededException`, `requests.exception.RequestException`
+
+
+
+#### create_transition(transition_type: str, \*, in_schema: Optional[dict] = None, out_schema: Optional[dict] = None, parameters: Optional[dict] = None, \*\*optional_args)
+Creates a transition handle, calls the POST /transitions endpoint.
+
+```python
+>>> import json
+>>> from pathlib import Path
+>>> from las.client import Client
+>>> client = Client()
+>>> in_schema = {'$schema': 'https://json-schema.org/draft-04/schema#', 'title': 'in', 'properties': {...} }
+>>> out_schema = {'$schema': 'https://json-schema.org/draft-04/schema#', 'title': 'out', 'properties': {...} }
+>>> # A typical docker transitions
+>>> docker_params = {
+>>>     'imageUrl': '<image_url>',
+>>>     'credentials': {'username': '<username>', 'password': '<password>'}
+>>> }
+>>> client.create_transition('docker', in_schema=in_schema, out_schema=out_schema, params=docker_params)
+>>> # A typical manual transitions
```

Comment:
What about 'A manual transition with UI' to give som context to the jsRemoteComponent part?

Reply:
This is autogenerated from the python-sdk, so comments on this file should be done there :P